tutorials/2d/2d_lights_and_shadows.rst

```diff
@@ -6,238 +6,326 @@
 Introduction
 ------------
 
-This tutorial explains how the 2D lighting works in the
-`lights and shadows <https://github.com/godotengine/godot-demo-projects/tree/master/2d/lights_and_shadows>`_ demo project.
-It begins with a brief description of the resources used in the final demo and then describes how
-to make a scene like the demo step by step.
+By default, 2D scenes in Godot are unshaded, with no lights and shadows
+visible. This means brightness is equal in all parts of the scene.
```

Comment:
I'm not sure that brightness is a good word either. Brightness is a perceptual quality that indicates the relative intensity of a light source. In 3D graphics we often use it to talk about the intensity of a light source, or the perceived intensity of light in a scene. But in 2D brightness is not necessarily controlled by light sources. A scene made up of all white sprites will be very bright compared to a scene with all black sprites. Since brightness is determined by the color of objects in your scene, it is not equal in all parts of the scene (unless your scene is a single static color).

I would just remove the second sentence and merge the first two paragraphs.